2020-04-11 17:20:58,957 ERROR: Exception on /device/2 [GET] [in c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\bkettle\OneDrive\development\proj\homie\app\routes.py", line 154, in device
    return render_template('view_device.html', user=current_user, current_device=current_device, title='Bitcoin Monthly Price in USD', max=17000, labels=line_labels, values=line_values)
NameError: name 'line_labels' is not defined
2020-04-11 17:21:08,538 ERROR: Exception on /device/2 [GET] [in c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\bkettle\OneDrive\development\proj\homie\app\routes.py", line 154, in device
    return render_template('view_device.html', user=current_user, current_device=current_device, title='Bitcoin Monthly Price in USD', max=17000, labels=line_labels, values=line_values)
NameError: name 'line_labels' is not defined
2020-04-11 17:23:51,836 ERROR: Exception on /device/2 [GET] [in c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\bkettle\onedrive\development\proj\homie\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\bkettle\OneDrive\development\proj\homie\app\routes.py", line 154, in device
    return render_template('view_device.html', user=current_user, current_device=current_device, title='Bitcoin Monthly Price in USD', max=17000, labels=line_labels, values=line_values)
NameError: name 'line_labels' is not defined
2020-04-11 17:29:25,141 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 17:53:48,574 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 17:57:02,539 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:04:10,741 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:04:39,582 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:49:09,415 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:49:49,229 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:50:14,067 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:51:26,105 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 18:52:25,379 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:01:06,863 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:01:44,440 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:02:11,121 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:04:20,209 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:05:12,687 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:05:58,489 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:06:19,335 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:06:49,539 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:07:06,739 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:07:38,857 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:08:04,791 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:22:31,817 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:25:08,409 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:26:46,380 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:29:53,505 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:31:39,669 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:34:19,613 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:47:29,891 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:47:55,978 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:51:29,097 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
2020-04-11 19:53:11,047 INFO: Homie startup [in C:\Users\bkettle\OneDrive\development\proj\homie\app\__init__.py:51]
